mmaction/models/heads/tsm_head.py

```diff
@@ -73,13 +73,16 @@ def init_weights(self):
         """Initiate the parameters from scratch."""
         normal_init(self.fc_cls, std=self.init_std)
 
-    def forward(self, x, num_segments):
+    def forward(self, x, num_segs):
         """Defines the computation performed at every call.
 
         Args:
             x (torch.Tensor): The input data.
-            num_segments (int): Number of frame segments. Default: 8.
-
+            num_segs (int): Useless in TSMHead. By default, `num_segs`
```

Comment:
Whether this could be removed, if it is useless? May discuss later.

Reply:
This argment could be removed if we add some codes in `recognizer2d.py`, codes like
```
if isinstance(self.cls_head, TSMHead):
    cls_score = self.cls_head(x)
else:
    cls_score = self.cls_head(x, num_segs)
```
Is this necessary? @innerlee 
Any better ideas? @dreamerlin

Reply:
I vote for keeping it unchanged and describe it in docstring as before